adapter: mz_objects includes cluster_id (#28255)
MaterializeInc/materialize: doc/user/content/sql/system-catalog/mz_catalog.md

`name`      | [`text`]             | The name of the object.
`type`      | [`text`]             | The type of the object: one of `table`, `source`, `view`, `materialized-view`, `sink`, `index`, `connection`, `secret`, `type`, or `function`.
`owner_id`  | [`text`]             | The role ID of the owner of the object. Corresponds to [`mz_roles.id`](/sql/system-catalog/mz_catalog/#mz_roles).

> `cluster_id`| [`text`]             | The ID of the cluster where the source, materialized view, index or sink is installed, or null for other object types. Corresponds to [`mz_clusters.id`](/sql/system-catalog/mz_catalog/#mz_clusters).

Reviewer: ```suggestion
`cluster_id`| [`text`]             | The ID of the cluster maintaining the source, materialized view, index, or sink. Corresponds to [`mz_clusters.id`](/sql/system-catalog/mz_catalog/#mz_clusters). `NULL` for other object types.
```

`name`      | [`text`]             | The name of the relation.
`type`      | [`text`]             | The type of the relation: either `table`, `source`, `view`, or `materialized view`.
`owner_id`  | [`text`]             | The role ID of the owner of the relation. Corresponds to [`mz_roles.id`](/sql/system-catalog/mz_catalog/#mz_roles).

> `cluster_id`| [`text`]             | The ID of the cluster where the source or materialized view is installed, or null for other object types. Corresponds to [`mz_clusters.id`](/sql/system-catalog/mz_catalog/#mz_clusters).

Reviewer: ```suggestion
`cluster_id`| [`text`]             | The ID of the cluster maintaining the source, materialized view, index, or sink. Corresponds to [`mz_clusters.id`](/sql/system-catalog/mz_catalog/#mz_clusters). `NULL` for other object types.
```
